common/Makefile.common.mk

```diff
@@ -19,6 +19,12 @@ PROJECT ?= oceanic-guard-191815
 ZONE    ?= us-west1-a
 CLUSTER ?= prow
 
+############################################################
+# install git hooks
+############################################################
+INSTALL_HOOKS := $(shell find .git/hooks -type l -exec rm {} \; && \
+                         find common/scripts/.githooks -type f -exec ln -sf ../../{} .git/hooks/ \; )
+
```

Comment:
Does the `INSTALL_HOOKS` will be automatically executed or we need to call it somewhere?

Reply:
Otherwise Looks good.

Reply:
it is automatically installed as it eval a global variable using the $shell command.
I don't think we have to check if the result sent in INSTALL_HOOKS is successful or not.